src/libsyntax/ast.rs

```diff
@@ -1300,6 +1300,11 @@ impl LitKind {
         }
     }
 
+    /// Returns a `LitKind::ByteStr` from `content`.
```

Comment:
Why does this need to have a constructor method, specifically?

Reply:
removed